install-worker.sh

```diff
@@ -53,6 +53,8 @@ sudo yum install -y yum-utils device-mapper-persistent-data lvm2
 sudo amazon-linux-extras enable docker
 sudo yum install -y docker-17.06*
 sudo usermod -aG docker $USER
+sudo sed -i -- 's!ExecStart=/usr/bin/dockerd!ExecStart=/usr/bin/dockerd --host=fd:// $DOCKER_OPTS!g' /usr/lib/systemd/system/docker.service
```

Comment:
- `EnvironmentFile=-/etc/default/docker` is needed before `ExecStart`
- It seems `--host=fd://` does not work with the docker on the linux aws is using.

Reply:
Rather than manipulating the text in the unit (especially as the packaged copy, edited here, could be overwritten) this could be added to a drop in unit in `/etc/systemd/system/docker.service.d/eks-customizations.conf` (see [systemd manpages for details](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#id-1.14.3)).

```bash
install -m 644 -D /dev/null /etc/systemd/system/docker.service.d/eks-customizations.conf
cat > /etc/systemd/system/docker.service.d/eks-customizations.conf <<EOF
[Service]
# Unset the ExecStart for overriding
ExecStart=
ExecStart=/usr/bin/dockerd --host=fd:// $DOCKER_OPTS
EOF
```

Reply:
Could also drop the configuration into `/etc/docker/daemon.json`: https://docs.docker.com/v17.06/engine/reference/commandline/dockerd/#miscellaneous-options